indexclient/client.pyi

```diff
@@ -0,0 +1,20 @@
+from typing import Dict, Iterable, List, Protocol
```

Comment:
Did you create this file or did PyCharm? If it was pycharm, should we add a .gitignore?

Reply:
I created this. This is for typing, I do not think we should add it to .gitignore.
IDE will be able to recognize this when you `import client`.